tests/unit/test_charm.py

```diff
@@ -17,72 +17,105 @@
 
 logger = logging.getLogger(__name__)
 
-POD_IP = b"1.1.1.1"
-VALID_NRF_URL = "https://nrf:443"
-EXPECTED_CONFIG_FILE_PATH = "tests/unit/expected_udmcfg.yaml"
+CERTIFICATE = "whatever certificate content"
 CERTIFICATES_LIB = (
     "charms.tls_certificates_interface.v3.tls_certificates.TLSCertificatesRequiresV3"
 )
-STORED_CERTIFICATE = "whatever certificate content"
-STORED_CSR = "Whatever CSR content"
+CERTIFICATE_PATH = "support/TLS/udm.pem"
+CSR = "Whatever CSR content"
+CSR_PATH = "support/TLS/udm.csr"
+EXPECTED_CONFIG_FILE_PATH = "tests/unit/expected_udmcfg.yaml"
+HOME_NETWORK_KEY = "whatever home network key"
+HOME_NETWORK_KEY_PATH = "etc/udm/home_network.key"
+NAMESPACE = "whatever"
+POD_IP = b"1.1.1.1"
+PRIVATE_KEY = "whatever private key"
+PRIVATE_KEY_PATH = "support/TLS/udm.key"
+VALID_NRF_URL = "https://nrf:443"
 
 
-class TestCharm(unittest.TestCase):
+class TestCharm():
```

Comment:
```suggestion
class TestCharm:
```
The paranthesis are not necessary.